### PR TITLE
Issue #14 - Migrating from npm to yarn: done

### PR DESCRIPTION
**Pull request by myroslav-kolomiiets**
rm package-lock.json, add yarn.lock and update README.md with instruction of installing project localy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,16 @@ MVP functionality:
...
 5. As a user, I want to have the list of my favorites movies
+
+# Installation and developing
+To install project run:
+
+`yarn install`
+
+To start dev server:
+
+`yarn run start`
```
Comment (zonzujiro) [inline README.md]: `yarn start` is ok

**Review by zonzujiro — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -16,3 +16,16 @@ MVP functionality:
 3. I want to see result while I typing the search request
 4. As a user, I want to open a page with all information about a movie, like related movies, genres, etc.
 5. As a user, I want to have the list of my favorites movies
+
+# Installation and developing
+To install project run:
+
+`yarn install`
```
Comment (zonzujiro) [inline README.md]: just `yarn` will work

**Review by zonzujiro — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (myroslav-kolomiiets) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (myroslav-kolomiiets) [reply README.md]: Done

**Review by zonzujiro — APPROVED** [review]
:fire:

**Pull request merged**